packages/kbn-optimizer/src/worker/webpack.config.ts

```diff
@@ -50,6 +50,7 @@ export function getWebpackConfig(
     profile: worker.profileWebpack,
 
     output: {
+      hashFunction: 'sha1',
```

Comment:
Do other webpack configs need to be updated?  storybooks / kbn-ui-shared-deps-{src,npm}?

Reply:
Unsure about the other configs. During discussions with the Security team, we wanted to avoid as many changes as possible to build/tooling under FIPS mode because it is generally for development. Example `NODE_OPTIONS='' yarn kbn bootstrap` disables FIPS because some modules use non-compliant algorithms during their own build process , like `sharp`, and error the whole bootstrap.

So, I think we can leave it for now unless needed. Security will be triaging non-compliant algorithm usage throughout the code base with relevant teams and it will arise then. The other aspect is whenever we get on Webpack 5, it already uses FIPS compliant algorithms.

Reply:
Were there issues with this config?  IIRC it should be development only too.

Reply:
It blocks Kibana from starting in development mode, therefore blocking other teams from fixing FIPS issues in their code.